### PR TITLE
Redirect CustomTabs docs to WebFu

### PR DESCRIPTION
**Pull request by andreban**
- Adds a JS redirect on customtabs.html
- Adds meta robots noindex to customtabs.html
- Tries to configure redirects.json to trigger server redirects.

### CODE REVIEW

**Comment by andreban** [comment]
@petele the complement to the WebFundamentals update for the Custom Tabs docs.

**Comment by petele** [comment]
@jeffposnick - do you have the ability to merge this?

File: android/customtabs.html [hunk 1]
```diff
@@ -1,5 +1,10 @@
 {{+bindTo:partials.standard_multidevice_article}}
 
+<meta name="robots" content="noindex">
+<script>
+   window.location.replace('https://developers.google.com/web/android/custom-tabs/');
```
Comment (jeffposnick) [inline android/customtabs.html:5]: Any reason not to use https://css-tricks.com/snippets/html/meta-refresh/ instead?

**Comment by jeffposnick** [comment]
I do, but I don't have any idea as to how this actually get published...

File: android/customtabs.html (hunk 1 above)
Comment (andreban) [reply android/customtabs.html:5]: Not particularly - I just saw other places in the code were doing like this and figured there must be a reason: https://github.com/GoogleChrome/multi-device/blob/master/android/installtohomescreen.html

Comment (jeffposnick) [reply android/customtabs.html:5]: Okay, fair enough.

**Pull request merged**